src/services/attachmentService.js

```diff
@@ -3,6 +3,8 @@ import { generateOcsUrl } from '@nextcloud/router'
 import { showError, showSuccess } from '@nextcloud/dialogs'
 import { translate as t } from '@nextcloud/l10n'
 
+const FOLDER = 'Calendar'
```

Comment:
can we somehow make the folder configurable? I know a few cases where this folder will not work in the user root. (e.g. when it is read only). 

E.g. this might be a good example: https://github.com/nextcloud/photos/blob/master/src/components/Settings/PhotosLocationSettings.vue

Reply:
Ping @sazanof ? :)

Reply:
> Ping @sazanof ? :)
```
PING @sazanof  (127.0.0.1) 56(84) bytes of data.
64 bytes from @sazanof  (127.0.0.1): icmp_seq=1 ttl=247 time=500000 ms
```
Yes, I looked at how it is implemented in the Photo application. In fact, we can add routes on the backend and expand the configuration on the frontend. I'll do it! :100: 

PS: Pings are unpredictable now.

Reply:
https://github.com/nextcloud/calendar/pull/4251/commits/ee408dc0a974625b8320203e1db4e16d21ccb1b8

Well, it was difficult, but that's in the past.